libevmasm/Assembly.h

```diff
@@ -83,6 +83,7 @@ class Assembly
 	template <class T> Assembly& operator<<(T const& _d) { append(_d); return *this; }
 	AssemblyItems const& items() const { return m_items; }
 	AssemblyItem const& back() const { return m_items.back(); }
+	void pop_back() { m_items.pop_back(); }
```

Comment:
If those cannot be used, I am not sure how to solve this. Perhaps introducing an actual switchtable instruction would be the best?